dashboard/app/models/pd/application/application_base.rb

```diff
@@ -110,6 +110,7 @@ class ApplicationBase < ApplicationRecord
     before_create -> {self.status = :unreviewed}
     after_initialize :set_type_and_year
     before_validation :set_type_and_year
+    validate :status_is_valid_for_application_type
```

Comment:
just curious if you moved this for a particular reason, e.g. does this change the order in which the validations run in a way that is important? I also noticed that it looked like this had some comments attached to it at its old location, so if you do want to move it, then those should come with it.